medicines/doc-index-updater/src/create_manager/search_index.rs

```diff
@@ -0,0 +1,31 @@
+use crate::{
+    create_manager::models::{BlobMetadata, IndexEntry},
+    search_client,
+};
+
+//use std::time::{Duration, Instant};
+
+pub async fn add_to_search_index(
+    search_client: &search_client::AzureSearchClient,
+    blob_name: &str,
+    blob: BlobMetadata,
+    file_size: usize,
+) -> Result<(), anyhow::Error> {
+    let storage_account =
+        std::env::var("STORAGE_ACCOUNT").expect("Set env variable STORAGE_ACCOUNT first!");
+    let container_name =
+        std::env::var("STORAGE_CONTAINER").expect("Set env variable STORAGE_CONTAINER first!");
+    let storage_path = format!(
+        "https://{}.blob.core.windows.net/{}/{}",
+        &storage_account, &container_name, &blob_name
+    );
```

Comment:
This is fine, but is there a way to get `create_blob` to return this URL along with the blob name?

Reply:
Great idea, I've gone pretty far with it. There's now a `Blob` which has everything we need for adding to the index.